static/sidebar/js/libs/utils.js

```diff
@@ -28,10 +28,21 @@
       };      
     };
 
+    this.trimTitle = function(data,limit){
+      return data = data.length > limit ? data.substring(0, limit) + "..." : data;
+    };
+
+    this.trimPrice = function(price){
+      (typeof price !== "undefined" && price !== null) ? price:""; 
```

Comment:
will remove it, i wanted to check if the argument came as undefined or null, but i think it's just not necessary.